package.json

```diff
@@ -365,7 +366,6 @@
     "@metamask/user-operation-controller": "^13.0.0",
     "@metamask/utils": "^8.2.1",
     "@ngraveio/bc-ur": "^1.1.12",
-    "@noble/ciphers": "^0.5.2",
```

Comment:
This package was only used as part of our controllers. Now that we are using these shared controllers, we can remove this dep.